hathor/cli/main.py

```diff
@@ -78,6 +79,7 @@ def __init__(self) -> None:
         self.add_cmd('tests', 'gen_twin_tx', twin_tx, 'Generate a twin transaction from a transaction hash')
         self.add_cmd('wallet', 'gen_kp_wallet', wallet, 'Generate a new KeyPair wallet')
         self.add_cmd('wallet', 'gen_hd_words', generate_valid_words, 'Generate random words for HD wallet')
+        self.add_cmd('wallet', 'gen_keys', generate_keys, 'Generate a private/public key pair and its address')
```

Comment:
Should we create a `side-dags` section?

Reply:
I got it. We might even rename it to `gen_side_dag_keys`. Even though generating keys is a generic task, this one is focused on generating keys for side dags. A remark about generic commands is that they are usually hard to maintain because we need to keep compatibility and they quickly become useless or hard to use for specific cases.

Reply:
Sure, I renamed it to `gen_poa_keys` in https://github.com/HathorNetwork/hathor-core/pull/1041/commits/7ba3d0fe840d424e4f83c6d2434ff228108d405c

Reply:
Thanks!